src/services/MultisigService.ts

```diff
@@ -94,9 +94,9 @@ export class MultisigService {
                 return (addressesFromNextLevel = addressesFromNextLevel.concat([
                     {
                         address: term.accountAddress,
-                        multisig: true,
+                        multisig: !!term.cosignatoryAddresses.length ? true : false,
```

Comment:
I think the ternary operator is redundant. We  can just use `multisig: !!term.cosignatoryAddresses.length`